nav2_costmap_2d/include/nav2_costmap_2d/obstacle_layer.hpp

```diff
@@ -131,37 +140,45 @@ class ObstacleLayer : public CostmapLayer
    * @param max_x
    * @param max_y
    */
-  virtual void raytraceFreespace(const nav2_costmap_2d::Observation & clearing_observation,
-      double * min_x, double * min_y,
-      double * max_x,
-      double * max_y);
-
-  void updateRaytraceBounds(double ox, double oy, double wx, double wy, double range,
-      double * min_x, double * min_y,
-      double * max_x,
-      double * max_y);
+  virtual void raytraceFreespace(
+    const nav2_costmap_2d::Observation & clearing_observation,
+    double * min_x, double * min_y,
+    double * max_x,
+    double * max_y);
+
+  void updateRaytraceBounds(
+    double ox, double oy, double wx, double wy, double range,
+    double * min_x, double * min_y,
+    double * max_x,
+    double * max_y);
 
   std::vector<geometry_msgs::msg::Point> transformed_footprint_;
   bool footprint_clearing_enabled_;
-  void updateFootprint(double robot_x, double robot_y, double robot_yaw, double * min_x,
-      double * min_y,
-      double * max_x,
-      double * max_y);
-
-  std::string global_frame_;  ///< @brief The global frame for the costmap
-  double max_obstacle_height_;  ///< @brief Max Obstacle Height
-
-  laser_geometry::LaserProjection projector_;  ///< @brief Used to project laser scans into point clouds
-
-  std::vector<std::shared_ptr<message_filters::SubscriberBase> > observation_subscribers_;  ///< @brief Used for the observation message filters
-  std::vector<std::shared_ptr<tf2_ros::MessageFilterBase> > observation_notifiers_;  ///< @brief Used to make sure that transforms are available for each sensor
- 
-  std::vector<std::shared_ptr<nav2_costmap_2d::ObservationBuffer> > observation_buffers_;  ///< @brief Used to store observations from various sensors
-  std::vector<std::shared_ptr<nav2_costmap_2d::ObservationBuffer> > marking_buffers_;  ///< @brief Used to store observation buffers used for marking obstacles
-  std::vector<std::shared_ptr<nav2_costmap_2d::ObservationBuffer> > clearing_buffers_;  ///< @brief Used to store observation buffers used for clearing obstacles
+  void updateFootprint(
+    double robot_x, double robot_y, double robot_yaw, double * min_x,
+    double * min_y,
+    double * max_x,
+    double * max_y);
+
+  std::string global_frame_;  /// @brief The global frame for the costmap
+  double max_obstacle_height_;  /// @brief Max Obstacle Height
+
+  // Used to project laser scans into point clouds
```

Comment:
Many of these comments (but not all) have been changed from doxygen comments to regular comments. We should be consistent.

If you make the comment of the form

```
/// @brief blah blah
type variable;
```
then it will still be a doxygen comment that applies to the variable.

Reply:
I wasn't sure whether putting the comment above the variable as a doxygen comment would apply to that variable.

Reply:
As a rule of thumb, doxygen comments by default apply to the thing right after them, although you can name the specify thing you want to refer to instead.

The `<` used in some comments is special marker allowing end of line comments. It indicates doxygen should apply the comment to the thing before the comment. The `<` is probably meant to represent an arrow pointing to what you are describing.